### PR TITLE
Use tailwind css to design navbar and import react icons

### LINKED ISSUE #33: Design nav bar
We need to design the nav bar. 

AC: 

- Create a navigation bar with the list and add an item, using design in [Figma](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44).
- Use text and icon for the menu.

### PR DESCRIPTION
**Pull request by hellodeborahuk**
## Description

- Using Tailwind CSS classes, I added a yellow background, as per the figma design. 
- Added icons for shopping list and add an item from react-icons and kept the words for clarity. 
- Centered the navbar by adding `flex` to the container div in `App.js`.

## Related Issue

Closes #33

## Acceptance Criteria

- [x] Create a navigation bar with the list and add an item, using design in [Figma](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44).
- [x] Use text and icon for the menu.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

From your terminal, pull down this branch with `git pull origin dd-design-navbar` and check that branch out with `git checkout dd-design-navbar`.
Then `npm start` to launch the app.
Check the design and navigate pages to test it still works as it should.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -8,13 +9,13 @@ const Navbar = () => {
...
-        <ul className="links">
+      <nav className="bg-yellow-200 fixed w-full bottom-0">
+        <ul className="flex justify-center gap-32 py-1 ">
           <li>
-            <NavLink to="/list">List</NavLink>
+            <NavLink to="/list" className="flex flex-col items-center"><TiShoppingCart className="text-4xl" /><span>List</span></NavLink>
           </li>
           <li>
-            <NavLink to="/add-item">add an item</NavLink>
+            <NavLink to="/add-item" className="flex flex-col items-center"><TiPlusOutline className="text-4xl" /><span>Add Item</span></NavLink>
```
Comment (hellodeborahuk) [inline src/components/Navbar.js:18]: Tried to use active class here rather than in css file, but I couldn't get it to stay after being clicked.

Comment (redapy) [reply src/components/Navbar.js:18]: The `active` class is generated by `<NavLink />` so I don't think we can do with tailwind. it's fine to stay in the CSS file I guess.

**Review by redapy — COMMENTED** [review]
Great job Debbie! LGTM

File: src/components/Navbar.js (hunk 1 above)
Comment (JimeBlue) [reply src/components/Navbar.js:18]: That´s right. Using the CSS file makes sense in this case. Wonderful job Debbie! It´s great that you included before and after images 😄

Comment (hellodeborahuk) [reply src/components/Navbar.js:18]: Thank you both.

**Pull request merged**